src/ServiceControl.Config/UI/InstanceAdd/ServiceControlAddAttachment.cs

```diff
@@ -53,7 +53,7 @@ async Task Add()
 
             var serviceControlNewInstance = viewModel.InstallErrorInstance ? ServiceControlNewInstance.CreateWithDefaultPersistence() : null;
 
-            if (viewModel.InstallAuditInstance)
+            if (viewModel.InstallErrorInstance)
```

Comment:
Was this a bug?

Reply:
Yes. I'm not sure when this was introduced/refactored though. [The version on the current release branch](https://github.com/Particular/ServiceControl/blob/release-4.32/src/ServiceControl.Config/UI/InstanceAdd/ServiceControlAddAttachment.cs#L54) has a bit different syntax and probably when the refactoring was done the issue was introduced.